crates/continuations/src/lib.rs

```diff
@@ -92,7 +184,9 @@ impl From<State> for i32 {
 /// TODO
 #[repr(C)]
 pub struct ContinuationObject {
-    pub parent: *mut ContinuationObject,
+    pub limits: StackLimits,
```

Comment:
We should optimise this such that it is only present in debug builds.
```suggestion
    pub limits: StackLimits, // TODO(dhil): Omit from release builds.
```

Reply:
I don't think that's a good idea. It would mean that in release mode, you can never have any backtraces at all anymore. That's not in line with what wasmtime currently does. Instead, we could try to respect wasmtime's configuration options regarding back traces, but that happens at runtime.
We may however have a special compile time flag to disable the backtrace mechanism completely, since the current design will have some overhead at every stack switch, which will improve in the future once we move away from the libcall-centric apprpoach.

Reply:
I'd be fine with a compile-time flag initially for experiments.

Reply:
The issue is that we will be using the same `StackLimits` machinery for preserving and restoring stack limits to enable stack overflow checks. Thefroe, this new flag may or may not disable stack limit checks as well. I suggest deferring the addition of this new flag until we have restored some degree of stack limit checks, some of which subsequently happens in this series of PRs.

Reply:
I guess we can have a "disable_backtraces" compile time flag which omits the reads/writes, and print no backtrace, for benchmarking.

Reply:
Let's do that as a follow-up PR